crates/hir/src/code_model.rs

```diff
@@ -743,6 +743,22 @@ impl Function {
         db.function_data(self.id).name.clone()
     }
 
+    /// Get this function's return type
+    ///
+    /// The returned type can be converted to a [`Type`] via its `ty`
+    /// method.
+    pub fn ret_type(self, db: &dyn HirDatabase) -> RetType {
```

Comment:
hmm I don't really see the point of wrapping the type in a `RetType` struct instead of just returning `Type` directly

Reply:
(`Param` has some justification since it could get e.g. a `name` method in the future)

Reply:
Ah, that's why... Do you want me to remove it ?

Reply:
I think that'd be better.

Reply:
Done !